ci/python_test.sh

```diff
@@ -14,28 +14,32 @@ function create_venv()
         source "$VENV/bin/activate"
         set -u
 
-        which pip
-
-        pip install --upgrade pip setuptools wheel
+        $VENV/bin/pip install --upgrade pip setuptools wheel
     fi
 }
 
 # in tree install
 create_venv
-pip install .
-pip install -r tests/requirement_tests.txt
 
-CURRENT=$(pwd)
-(cd .. && pytest ${CURRENT}/tests)
+$VENV/bin/pip -v install --force .
+$VENV/bin/pip -v install -r tests/requirement_tests.txt
+
+
+pushd $(pwd)/tests
+pytest .
+popd
+
 deactivate
 
 # sdist install
 DIST_DIR="$VENV/dist"
 mkdir -p "$DIST_DIR"
 create_venv
+
 python3 setup.py sdist --dist-dir "$DIST_DIR"
```

Comment:
For future reference, I think this is becoming deprecated, see [this](https://github.com/pytest-dev/pytest/issues/9225), pointing to [this](https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html).

Reply:
Nice! I switched to using that.